azure-pipelines.yml

```diff
@@ -26,9 +26,6 @@ parameters:
     type: object
     default:
       - 'hudi-common'
-      - 'hudi-flink-datasource/hudi-flink'
-      - 'hudi-flink-datasource/hudi-flink1.13.x'
-      - 'hudi-flink-datasource/hudi-flink1.14.x'
   - name: job2Modules
```

Comment:
What is the effect of removing these 3 items ?

Reply:
reverted